examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<ul><li>I don't think the <code>handler</code>  need to be defined again</li><li>I don't see you're setting the <code>password</code>  secret on the project anywhere, so this will fail right ?</li><li>I suggest to uncomment the example that specify specific secrets, and remove the example with empty list (or if that would be the first example, add the comment regarding the empty list here)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>ok</p>

Reply:
And the `password` is set in the previous cell. The function running on the 2nd project will show all 3 secret values defined for the project.